README.md

```diff
@@ -29,18 +29,14 @@ You can either use it from your local machine OR using docker.
 ### Phel code
 
 1. Write your phel code in `src/`
-1. Execute your code with `./vendor/bin/phel run src/command/main.phel`
-1. Or with a PHP server with `php -S localhost:8000 -t ./src/web`
+2. Execute your code with `./vendor/bin/phel run src/command/main.phel`
+3. Or with a PHP server with `php -S localhost:8080 src/web/index.php`
```

Comment:
With the bug fix I just did, we can change this back to the original value. Then the images and css files are loading fine.